src/app/pcpolicy/pcpolicy.component.scss

```diff
@@ -0,0 +1,13 @@
+.pcpolicybody{
+	h1{
```

Comment:
Couldn't find any other way to get some free space between the header and paragraph text. I have checked for responsiveness and It is responsive. other solution I thought was adding 2-3 <br> tags but that would not be neat.